Softeq.XToolkit.Bindings.Tests/StubProducer.cs

```diff
@@ -0,0 +1,16 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+
+namespace Softeq.XToolkit.Bindings.Tests
+{
+    public class StubProducer
+    {
+        public event EventHandler SimpleEvent;
+        public event EventHandler<string> GenericStringEvent;
+
+        public virtual void RaiseSimpleEvent() => SimpleEvent?.Invoke(this, EventArgs.Empty);
```

Comment:
NSubstitute also has functionality to simulate events on stubs
https://nsubstitute.github.io/help/raising-events/

Reply:
Can't be used, because NSubstitute has constraint for `EventHandler<TEventArgs>` event:

```cs
EventHandlerWrapper<TEventArgs> EventWith<TEventArgs>(TEventArgs eventArgs) where TEventArgs : EventArgs
```

Also, using `Raise`, in this case, looks more redundant.